src/components/timeline/index.js

```diff
@@ -64,7 +64,8 @@ module.exports = {
             let filteredSharedItems = [];
             for(var i=0; i < items.length; i++) {
                 let currentSharedItem = items[i];
-                if (!currentSharedItem.path.startsWith("/" + currentSharedItem.owner + "/.profile/")) {
+                if (!currentSharedItem.path.startsWith("/" + currentSharedItem.owner + "/.profile/")
+                    && !currentSharedItem.path.startsWith("/" + currentSharedItem.owner + "/shared/.")) { //groups
```

Comment:
do you think this would be better to fix by making the dirs hidden?

Reply:
I think the setup is fine. The timeline will not display hidden files.  See createTimelineEntry().  The use of this filter method is to short-circuit the gathering of results (expensive operation) so it doesn't even try to createEntries for these items.